script/release.sh

```diff
@@ -139,13 +139,20 @@ CONTRIBUTORS=$(git log "${LAST_RELEASE_VERSION}".. --format="%aN" --reverse | so
 CHANGELOG=$(git log "${LAST_RELEASE_VERSION}".. --oneline | grep -v 'Merge pull request')
 
 CHECKSUM=""
-for file in $(ls bin/docker-machine*); do
+for file in $(ls bin/docker-machine-*); do
   SHA256=$(openssl dgst -sha256 < "${file}")
   MD5=$(openssl dgst -md5 < "${file}")
   LINE=$(printf "\n * **%s**\n  * sha256 \`%s\`\n  * md5 \`%s\`\n\n" "$(basename ${file})" "${SHA256}" "${MD5}")
   CHECKSUM="${CHECKSUM}${LINE}"
 done
 
+# Generate Checksum file
+(
```

Comment:
`cd` into the `bin` directory will make the result more clean:

With `cd bin`, the generated `sha256sum.txt` is:

```
efc5306920194c3ac4be3baaa09b4cbb398db0e6e6afe0e241287b3c4db9dcfe  docker-machine-Darwin-x86_64
87bc42fa15e1fdc1d543256882828eb6bc7728f2cc66246c2aa33bef3efdb41a  docker-machine-Linux-armhf
74294a6593310bb273c99a915bf95fa5990a995dbbbc9f6cd0b26cfb640ce43c  docker-machine-Linux-x86_64
f91971878808192da79ffc6cf25fb5aa43df9441d8fb4c40979552e81c5a8bf2  docker-machine-Windows-i386.exe
c8a07bc01348bb69810f63a5f39d2dbfa0376037dbcaa9d4f45431419254406a  docker-machine-Windows-x86_64.exe
```

Without `cd bin`, the result is:

```bash
f80862f0f0fe77f756ad49794b61be5cf14061095e4233f65c23ea4ca63a0c29  bin/docker-machine-Darwin-x86_64
b219f94aea6bac213a257c4e14df9609b4766646573193b67e1778feacc08742  bin/docker-machine-Linux-armhf
b77ddd4d7f47902c6082375962caa454e9466e2e484ed6cc1dea08da50d4e285  bin/docker-machine-Linux-x86_64
83092b781f8b818cee23ad2f93949de88689b986aa3ed15d060d5c069fef1b15  bin/docker-machine-Windows-i386.exe
ac5180b920ad97e1dd18a03cf221f84b42bb0038e0a1871de8bbefd6d2a849ea  bin/docker-machine-Windows-x86_64.exe
```

There will be a `bin/` path prefixed on the filename in the checksum file by default. Although the prefix can be removed later, it has to be done on both checksum files, so `cd bin` may be a cleaner way.